### ISSUE TITLE
Angular directives not working when defined in click event function

### ISSUE THREAD

**Issue by Ocajian** [pablojim/highcharts-ng#255]
Hi There

In my angular controller named "SentimentChartController" I have set up a click event on my highchart graph nodes as follows:

    $scope.AveSentimentChartConfigMain.options.plotOptions.series.point.events.click = function () {
            var containerDiv = document.createElement('div');
            containerDiv.setAttribute('id', 'postListPopUpContainer');
            var contentDiv = document.createElement('div');
            contentDiv.setAttribute('id', 'postListPopUp');
            contentDiv.innerHTML = '<div ng-controller="SentimentChartController"><img ng-click="closeDrillDown()" src="images/closePostListPopUp.svg" title="Click to close" class="closeIcon" width="18" height="18"><h2>' +
            this.category + ' ' + this.series.name + '</h2>' +
            '<p class="drillDownInfo"><strong>' + this.y + '%</strong> of posts where we can determine a sentiment are ' + this.category + '</p></div>';
            document.body.appendChild(containerDiv);
            containerDiv.appendChild(contentDiv);
        };

However none of the angular functionality that is declared inside of this click event works.

For example when I click on the following image:

    <img ng-click="closeDrillDown()" src="images/closePostListPopUp.svg" title="Click to close" class="closeIcon" width="18" height="18">

My function which I defined in the same controller does not get triggered:

    $scope.closeDrillDown = function() {
          console.log('the function is working');
        };

I read somewhere else that anything placed inside a chart click event will be "outside of the angular scope and needs to be added with $scope.$apply() but unfortunately I am unsure of where / how to do this in relation to my problem.

**Comment by Ocajian** [comment]
This is the thread where you mention using wrapping a click event with an apply - https://github.com/pablojim/highcharts-ng/issues/56

**Comment by pablojim** [comment]
Could you make a simplified jsfiddle that shows the problem?

**Comment by Ocajian** [comment]
Hi There

Here is a simple jsfiddle as requested - http://jsfiddle.net/47m46/8/

I have set it up to display content below the graph once you click on a series node/point

You will then see a button followed by text.

The button has an ng-click directive on it but the function I call in the ng-click does not get triggered when you click on it. I have also noticed that no angular stuff works if placed inside my highcharts click function.

So my question is how do I set-up my highcharts click function to be able to use angular directives such as ng-click or ng-repeat?
